Bylaw 7.tex

```diff
@@ -121,25 +122,15 @@ \section{Finances}
 	&&& A Discipline Club is required, upon the request of the Board of Directors, to release a current budget and up-to-date record of financial transactions within thirty (30) days of such a request.
 \end{easylist}
 
-\section{Track One Committee}
+\section{TrackOne Committee}
 \begin{easylist}
 \ListProperties(Start2=0)
-&& Governing Regulations
-	&&& The TrackOne Committee shall be chaired by the TrackOne Class Representative.
-	&&& The organizational structure of the TrackOne Committee is outlined in the TrackOne Committee Constitution.
-	&&& Financial support for the TrackOne committee shall be allocated annually in the amount of \$250 + \$4.50 per student. If the Engineering Society increases membership fees to adjust for cost of living, as per Bylaw 1 Section 1.2.4, then the funding for the TrackOne Committee shall be adjusted by the same percentage.
-	&&& Financial support shall be delivered in the form of three cheques.
-	&&& TrackOne Committee funding shall be withheld until its release by the Board. The Board may decline to release funding for the following reasons:
-		&&&& The TrackOne Committee has pranks from previous years which are in a state of disrepair.
-		&&&& The TrackOne Committee does not have a constitution that has been ratified by the Board or is not in compliance with its Board-approved constitution.
-		&&&& The TrackOne Committee does not have an operating budget that has been approved by the Finance Committee.
-		&&&& The TrackOne Committee has not run elections in compliance with the Society’s Bylaws.
-			&&&&& The TrackOne Committee Constitution shall clearly state that the TrackOne Representative must be elected through the Engineering Society, barring extraneous circumstances, as defined in Bylaw 3.
-	&&& A two-thirds majority of members of the Board of Directors, not of the discipline in question, at a meeting of the Board of Directors shall be required to release a clubs funding.
-	&&& The TrackOne Committee is required, upon the request of the Board of Directors, to
-release a current budget and up-to-date record of financial transactions within thirty
-(30) days of such a request.
-
+&& Exceptions to Chapter 1 for the TrackOne Committee
+	&&& The per-discipline funding, as described in Section 1.0.1, shall be \$250 instead of \$1000.
```

Comment:
```suggestion
	&&& The per-discipline funding, as described in Section 1.0.1, shall be one quarter of the amount specified in that section (for example, \$250 instead of \$1000).
```